app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -138,6 +138,10 @@ def biometric_comparison_needed?
         !current_user.identity_verified_with_biometric_comparison?
     end
 
+    def enhanced_ipp_required?
```

Comment:
What is this method doing on the authorization controller? I'm not sure I see where it is used.